### PR TITLE
Activities/iconlist

### PR DESCRIPTION
**Pull request by malin-mallan**
List of more icons for the activity screen. 

Adds a theme with custom propeties to unify icon sizes across the app.

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -6,9 +6,39 @@ import { NavigationContainer } from '@react-navigation/native';
...
+  colors: {
+    ...DefaultTheme.colors,
+
+  },
+  iconSizes: {
+    small: 30,
+    medium: 40,
+    large: 60
+  }
+};
```
Comment (EMattfolk) [inline App.tsx]: I feel like the theme should be in a different file somewhere.

File: src/components/ActivityScreen.tsx [hunk 2]
```diff
@@ -30,18 +25,19 @@ const IconSettingsView = () => (
...
   );
 }
 
 const ViewContent = ({ navigation }: any) => {
-  let navigationButtonSize = 40;
-
   const { colors } = useTheme();
 
+  const [visible, setVisible] = React.useState(false);
+  let navigationButtonSize = 40;
```
Comment (EMattfolk) [inline src/components/ActivityScreen.tsx]:
```suggestion
  const navigationButtonSize = 40;
```

**Review by EMattfolk — APPROVED** [review]
Looks good to me!

**Review by augustcarlson — APPROVED** [review]
Looks good!

**Pull request merged**